projectDocs/dev/createDevEnvironment.md

```diff
@@ -11,48 +14,54 @@ After forking the repository into your user account (`YOUR-USERNAME`), clone wit
 git clone --recursive https://github.com/YOUR-USERNAME/nvda.git
 ```
 
-
 ### Keeping the fork in sync
+
 When you fork the repository, GitHub will create a copy of the master branch.
-However, this branch will not be updated when the nvaccess master branch is updated.
-To ensure your work is always based on the latest commit in the nvaccess master branch, it is recommended that your master branch be linked to the nvaccess master branch, rather than the master branch in your GitHub fork.
+However, this branch will not be updated when the NV Access master branch is updated.
+To ensure your work is always based on the latest commit in the NV Access master branch, it is recommended that your master branch be linked to the NV Access master branch, rather than the master branch in your GitHub fork.
```

Comment:
```suggestion
To ensure your work is always based on the latest commit in the `nvaccess/nvda` master branch, it is recommended that your master branch be linked to the `nvaccess/nvda` master branch, rather than the master branch in your GitHub fork.
```